src/java.base/share/classes/java/util/PriorityQueue.java

```diff
@@ -209,6 +209,25 @@ else if (c instanceof PriorityQueue<?>) {
         }
     }
 
+    /**
+     * Creates a {@code PriorityQueue} containing the elements in the
+     * specified collection. The {@code PriorityQueue} will order its
+     * elements according to the specified comparator.
```

Comment:
The wording "will order its elements" is consistent with the other constructors but I wonder if "its" might be mis-read to mean the collection rather than the queue. If you change it to "will order the priority queue's elements" will avoid that.

You'll need to add `@since 23`. Also new tests will need to be added for this constructor.

Reply:
makes sense, updated it.